test/testrunner.py

```diff
@@ -186,17 +187,23 @@ def pull_dlc_images(images):
         run(f"docker pull {image}", hide="out")
 
 
-def setup_eks_cluster(framework_name):
+def setup_eks_cluster(framework_name, is_neuron):
     frameworks = {"tensorflow": "tf", "pytorch": "pt", "mxnet": "mx"}
     long_name = framework_name
     short_name = frameworks[long_name]
-    codebuild_version = os.getenv('CODEBUILD_RESOLVED_SOURCE_VERSION')[0:7]
+    codebuild_version = os.getenv("CODEBUILD_RESOLVED_SOURCE_VERSION")[0:7]
     num_nodes = 1 if is_pr_context() else 3 if long_name != "pytorch" else 4
-    cluster_name = f"dlc-{short_name}-cluster-" \
-                   f"{codebuild_version}-{random.randint(1, 10000)}"
+    cluster_name = f"dlc-{short_name}-cluster-" f"{codebuild_version}-{random.randint(1, 10000)}"
```

Comment:
nit: Since this is a single line, both the strings can be joined into one, rather than breaking it.